doc/source/conf.py

```diff
@@ -11,10 +11,12 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
 import sys
+from pathlib import Path
 
-sys.path.insert(0, os.path.abspath("."))
+from sphinx.ext import apidoc
+
+sys.path.insert(0, str(Path.cwd().absolute()))
```

Comment:
Can we completely remove this line? @rodolfocarobene

Reply:
Yes definitely, I'm not sure why it is even there, but probably from some careless copy-pasting

Reply:
I believe that `sphinx-quickstart` generated it. It's just not very useful to us, because we always work with installed environments, even in development mode, so there is no need to manually add the path of the package.

I removed it :)